### ISSUE TITLE
Feature Request: sync to RSS Service (Feedly, Miniflux, Nextcloud News, etc).

### ISSUE THREAD

**Issue by MintMain21** [NeoApplications/Neo-Feed#16]
Occasional crash aside, this works great as a independent feed reader, but it'd be nice to have it sync with services like Feedly etc as well.

**Comment by citizenserious** [comment]
I agree on the long run it would be very awesome to be able to sync it with nextcloud  (:

**Comment by RuthlessRuler** [comment]
Don't forget Inoreader.
It would amazing if this can achieved.

**Comment by kajdo** [comment]
i think freshrss is very popular in the selfhosting community and im messing aroud with it currently, would be great to have a way to connect to it via neo feed as well

**Comment by Froggy232** [comment]
Hi,
Any news on this? I'm also very interested to link it to FreshRSS.
Thanks for your app, it's already very interesting in this state.

**Comment by Akruidenberg** [comment]
Me top!

**Comment by Alessio-Olivieri** [comment]
Is anybody working on this? I could try.

**Comment by saulhdev** [comment]
You can try @Alessio-Olivieri

**Comment by saulhdev** [comment]
Added in https://github.com/NeoApplications/Neo-Feed/commit/7c17a695f9288ba92a5c9da82212a0683e2335c4
